src/resolvers/__tests__/grid.test.js

```diff
@@ -185,7 +185,7 @@ describe('Grid', () => {
                 name: 'Lorem Ipsum'
               }]
             })
-
+            // TODO remove this parameter search from grids in favor of params
```

Comment:
this is for grids which are going to need to be updated in the future since ingest team is currently updating their schema

Reply:
I don't understand this TODO, is it just switching from the deprecated option to the params object? Or is something changing soon with grids? 

If something is changing we should get a ticket number to add to the comment.

Reply:
No this is just referring to using params object instead of the deprecated obj in the tests. I should write that ticket to update grid though now that the changes are in CMR and that's where I would make those test modifications.

Reply:
I think you can probably just remove the TODO, deprecated params are used all over the tests. A ticket to update them would be good